pkg/controller/certificaterequests/acme/acme_test.go

```diff
@@ -118,6 +147,19 @@ func TestSign(t *testing.T) {
 		t.FailNow()
 	}
 
+	ipCSRPEM := generateCSRWithIPs(t, sk, "10.0.0.1", nil, []string{"10.0.0.1"})
+	ipCSR, err := pki.DecodeX509CertificateRequestBytes(ipCSRPEM)
+	if err != nil {
+		t.Error(err)
+		t.FailNow()
```

Comment:
Nit:

```suggestion
		t.Fail(err)
```

...if you feel like it.

Reply:
Think you meant Fatal